### DISCUSSION TITLE
"Thin client / Server" setup with Sunshine and Moonlight

### DISCUSSION THREAD

**Discussion by gflash999** [LizardByte/community#479] [Sunshine]
Hello,

I did something similar back in my Windows days, and it worked nicely. I could use a remote PC via NVIDIA Gamestream and Moonlight like a gaming-enabled remote desktop. When on a trip, I could leave my heavy gaming laptop at home, and connect to it with my lightweigh fanless laptop, and game on it with 15-30fps (depending on the internet connection). Within my home network, even 60fps were no problem. That's enough for the types of games I play.

So I'm thinking about taking it one step further, this time with Sunshine instead of NVIDIA Gamestream:

- Put a gaming PC in my basement, make it wake-on-LAN compatible, and put a HDMI 4k Dummy Plug in it
- In my room, boot my weak fanless laptop, send out wakeup signal to PC, connect to it via Moonlight, make it fullscreen
- Basically forget that I'm not directly on my PC and do all the desktop and gaming stuff as I usually would if it was standing besides my desk
- If not at home, use VPN connection (handled by my router) to do basically the same

I have an NVIDIA 40XX Series graphics card and an Intel processor. I would use the Flatpack versions of both Moonlight and Sunshine.

Is anyone using such a setup and can tell me about their experiences?

**Comment by gschintgen** [comment]
It all depends on your needs.

Remote connections are not always fun. And there are some keyboard issues with non-US layouts (AltGr key can be problematic with Windows clients). The chroma 4:2:0 subsampling is also sometimes noticeable. But that is being worked on. Let's hope the 4:4:4 support is coming to Linux too.

For some basic PC management stuff from my phone (wake, suspend, reboot, sunshine restart, etc.) I improvised a NodeRed Dashboard. (I have it running on my NAS anyway.)

Just try for a few days.

I don't use my setup exclusively that way, but when I do I quickly get confused if the Gnome DE I'm seeing is the local one or the remote one... (Make sure Moonlight is passing through all hotkeys, like Alt-Tab...)

If there are Sunshine binaries for your distribution use those instead of the flatpak.

**Comment by Rivlin77** [comment]
I use parsec-vdd as a virtual monitor, the good thing is, if you use different pc / different resolution, when you connect you can easily switch resolution after connecting.
And yes, I pretty much do this. My gaming PC is window. I use Playnite if i am streaming to a TV (Nvidia shield / google tv / xbox S) laptop (win 11 surface pro 8, Fedora lenovo duet 3i, i use to take a old asus transformer t101ha).
Raspberry pi 5 is good too, right now i am trying to make it run on a orange pi Zero 2w.
I use mostly in house.

[a comment by an automated account is not shown]
